lib/open_mic.rb

```diff
@@ -0,0 +1,30 @@
+require 'pry'
+
+class OpenMic
+  attr_reader :location, :date, :performers
+
+  def initialize(info)
+    @location = info[:location]
+    @date = info[:date]
+    @performers = []
+  end
+
+  def welcome(user)
+    @performers << user
+  end
+
+  def repeated_jokes?
+    jokes = []
+    @performers.each do |performer|
+      performer.jokes.each do |joke|
+        jokes << joke
+      end
+    end
+   if jokes == jokes.uniq
```

Comment:
The logic here is great!  You could also end this method with `jokes != jokes.uniq`, which would accomplish the same thing as the `if` statement!